content/en/docs/concepts/architecture/control-plane-node-communication.md

```diff
@@ -11,62 +11,108 @@ aliases:
 
 <!-- overview -->
 
-This document catalogs the communication paths between the control plane (apiserver) and the Kubernetes cluster. The intent is to allow users to customize their installation to harden the network configuration such that the cluster can be run on an untrusted network (or on fully public IPs on a cloud provider).
-
-
+This document catalogs the communication paths between the control plane (the API server) and the
```

Comment:
Since we're touching this text, we should avoid implying that “control plane” == `kube-apiserver`. Instead, what we'd say is that the API server is commonly the only part of the control plane that communicates with components outside the control plane (the _hub and spoke_ API pattern).